src/drivers/general/queries/experimental/UnicodeStringFreed/UnicodeStringFreed.ql

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+/**
+ * @id cpp/drivers/unicode-string-not-freed
+ * @name Unicode String Not Freed
+ * @description UnicodeString objects created with RtlCreateUnicodeString must be freed with RtlFreeUnicodeString.
+ * @platform Desktop
+ * @security.severity Medium
+ * @feature.area Multiple
+ * @impact Exploitable Design
+ * @repro.text A UNICODE_STRING object is created with RtlCreateUnicodeString but not freed with RtlFreeUnicodeString.
+ * @owner.email sdat@microsoft.com
+ * @opaqueid CQLD-D0005
```

Comment:
CQLD-D0005 is used as the opaque ID for KeSetEventIrql - please update this to CQLD-D0006.  (General TODO for us: create an index of rules with their IDs and opaque IDs.)  You should also re-run and re-generate the test baseline SARIF file to pick up the change to opaque ID afterwards.